app/src/main/res/values-nl/strings.xml

```diff
@@ -100,6 +100,8 @@
     <!-- WaitActivity -->
     <string name="wait_message_partner_rhythm_tapping">Wachten tot het andere apparaat klaar is met ritme tappen…</string>
     <string name="wait_message_slave_verification_method">Wachten tot het andere apparaat een verificatie methode heeft gekozen…</string>
+    <string name="by_tapping_a_rhythm">Door een ritme te tikken</string>
+    <string name="by_tapping_on_an_image">Door op een plaatje te drukken</string>
```

Comment:
Ik denk niet dat dit helemaal de juiste vertaling is, misschien: "Door op een _aantal_ plaatjes te drukken"